emcache/protocol.py

```diff
@@ -341,6 +341,27 @@ async def get_and_touch_command(
         finally:
             self._parser = None
 
+    async def cache_memlimit_command(self, value: int, noreply: bool) -> Optional[bytes]:
+        noreply = b" noreply" if noreply else b""
+
+        data = b"cache_memlimit " + f"{value:d}".encode() + noreply + b"\r\n"
+
+        if noreply:
```

Comment:
This is a very gross mistake, and how my tests allowed it. I'm sorry. We are correcting it.

Reply:
Oops, it seems that b"" will return False. However, I will still add the extra variable. It will save me from such confusion.